Framework/PythonInterface/mantid/plots/helperfunctions.py

```diff
@@ -993,3 +995,20 @@ def line_colour_fill(ax):
             i = i + 1
 
     ax.get_figure().canvas.draw()
+
+
+# Update the color bar scale on a given image
+def update_colorbar_scale(figure, image, scale, vmin, vmax):
```

Comment:
Good code consolidation.

Could we add a documentation string describing the intent and arguments?